views/modify_schematisation_dialogs.py

```diff
@@ -275,7 +249,6 @@ def __init__(self, parent=None, command=None):
         self.spinbox_search_distance.setMinimum(2)
         self.spinbox_levee_distace.setMaximum(5000)
         self.spinbox_levee_distace.setMinimum(1)
-        self.setWindowTitle(_translate("self", "Create breach locations", None))
```

Comment:
~~We probably do want to set the windows title but no longer use the `_translate`:
`self.setWindowTitle("Create breach locations")`~~
Niet meer relevant, zie comment boven